xclim/indices/_multivariate.py

```diff
@@ -970,7 +978,7 @@ def precip_accumulation(
         pr = rain_approximation(pr, tas=tas, thresh=thresh, method="binary")
     elif phase == "solid":
         pr = snowfall_approximation(pr, tas=tas, thresh=thresh, method="binary")
-    pram = rate2amount(pr)
+    pram: xarray.DataArray = rate2amount(pr)
```

Comment:
Isn't the output of  `rate2amount` already annotated as `DataArray` ?

Reply:
`rate2amount` is decorated with a `_register_conversion` and the typing for that hasn't been addressed yet, so whatever comes out of that function is seen as `Any`. I'll see if I can tackle that.

Reply:
I can't seem to figure this one out. I'll make a note to return to this later on.

Reply:
This is due to the decorator issue. FIXME's have been added.